drawBot/aliases.py

```diff
@@ -0,0 +1,12 @@
+import os
+from typing import Sequence
+
+Point = Size = tuple[float, float]
```

Comment:
```suggestion
Point = tuple[float, float]
Size = tuple[float, float]
```
This makes mypy a lot happier.